brightnessd.c

```diff
@@ -1081,21 +1161,29 @@ static int parse_args(int len, char** args) {
     int opt = 0;
     static struct option long_options[] = {
         {"cycle-brightness",   required_argument,       0,  'c' },
+        {"cycle-command",      required_argument,       0,  'd' },
         {"timeout-brightness", required_argument,       0,  't' },
+        {"timeout-command",    required_argument,       0,  'u' },
         {"help",               no_argument,             0,  'h' },
         {0,                    0,                       0,  0   }
     };
 
     int long_index = 0;
-    while ((opt = getopt_long(len, args, "c:t:h",
+    while ((opt = getopt_long(len, args, "c:d:t:u:h",
                               long_options, &long_index)) != -1) {
         switch (opt) {
         case 'c':
             err = parse_uint8_t(optarg, &DIM_PERCENT_INTERVAL);
             break;
+        case 'd':
+            EXTERNAL_CMD_INTERVAL = optarg;
```

Comment:
Hm, it would be more explicit to do an `strdup{,a}` on `optarg`. However, a matter of taste though, I guess.